src/loader/PointAttributes.js

```diff
@@ -19,20 +19,20 @@ Potree.PointAttributeNames.NORMAL				= 10;
  * @class
  */
 Potree.PointAttributeTypes = {
-	DATA_TYPE_DOUBLE	: {ordinal : 0, size: 8},
-	DATA_TYPE_FLOAT		: {ordinal : 1, size: 4},
-	DATA_TYPE_INT8		: {ordinal : 2, size: 1},
-	DATA_TYPE_UINT8		: {ordinal : 3, size: 1},
-	DATA_TYPE_INT16		: {ordinal : 4, size: 2},
-	DATA_TYPE_UINT16	: {ordinal : 5, size: 2},
-	DATA_TYPE_INT32		: {ordinal : 6, size: 4},
-	DATA_TYPE_UINT32	: {ordinal : 7, size: 4},
-	DATA_TYPE_INT64		: {ordinal : 8, size: 8},
-	DATA_TYPE_UINT64	: {ordinal : 9, size: 8}
+	DATA_TYPE_DOUBLE: {ordinal: 0, size: 8},
```

Comment:
Is it possible to preserve whitespaces after linting in same cases, in order to allign enumerations?